addons/io_scene_gltf2/__init__.py

```diff
@@ -898,6 +1014,52 @@ def draw(self, context):
         layout.prop(operator, 'will_save_settings')
 
 
+class GLTF_PT_export_gltfpack(bpy.types.Panel):
+    bl_space_type = 'FILE_BROWSER'
+    bl_region_type = 'TOOL_PROPS'
+    bl_label = "gltfpack"
+    bl_parent_id = "FILE_PT_operator"
+    bl_options = {'DEFAULT_CLOSED'}
+
+    @classmethod
+    def poll(cls, context):
```

Comment:
Add a check to display only if the path is provided in preferences

Reply:
Also fixed. I hope this makes sense, I was able to reproduce both cases at least. 

One weird thing i noticed though is that poll seems to be run more than once (2 or 3 times) at all times... which seems weird, but is probably also just me not knowing how Blender calls various things and poll is one of those - I would assume.